tools/c7n_azure/tests/azure_common.py

```diff
@@ -115,7 +144,7 @@ def setUp(self):
         ThreadHelper.disable_multi_threading = True
 
         # Patch Poller with constructor that always disables polling
-        self.lro_patch = patch.object(msrest.polling.LROPoller, '__init__', BaseTest.lro_test_init)
+        self.lro_patch = patch.object(msrest.polling.LROPoller, '__init__', BaseTest.lro_init)
```

Comment:
test classes with 'test' in the method name trigger some test frameworks to treat this as a test which it it is not.